apps/services/contentful-entry-tagger/infra/contentful-entry-tagger-service.ts

```diff
@@ -0,0 +1,21 @@
+import { service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
+
+export const serviceSetup = (): ServiceBuilder<'contentful-entry-tagger-service'> =>
+  service('contentful-entry-tagger-service')
+    .image('contentful-entry-tagger')
```

Comment:
Please use the name as defined in `workspace.json`

Reply:
I think I've got matching names now